UserGuide.md

```diff
@@ -593,6 +596,26 @@ Once the GitHub action processes the comment and sends the message, it will be p
 ![Example GitHub Comment](images/Example_GitHub_Comment.png)
 
 
+### <a name="starting-a-pr-comparison-run-from-github"/> **Starting a PR Comparison Run from GitHub**
+
+There is another type of run that can be triggered in CARROT from GitHub and that is the PR Comparison Run.  A PR Comparison run triggers two runs of a test, one with a docker image generated from the head commit of the PR and one with a docker image generated from the base commit of the PR.  Triggering it is also exactly the same is triggering a single run:
+
+
+```
+#carrot_pr({test_name},  {test_docker_key}, {eval_docker_key})
+```
+
+
+where
+
+
+
+* `{test_name}` is the name of a test you’ve already defined in CARROT,
+* `{test_docker_key}` is the name of an input to your test WDL that accepts a Docker image created from your repository, and
```

Comment:
It seems kind of weird to have commas and `and`s in a bulleted list but it's not a big deal either way.

I might highlight test and eval in these two lines somehow because I had to read them about 4 times before I could tell how they were different.

Reply:
That's fair.  I'll make those changes before I merge.